rust/headless-client/src/windows.rs

```diff
@@ -98,10 +103,7 @@ fn fallible_windows_service_run(arguments: Vec<OsString>) -> Result<()> {
     tracing::info!(?arguments, "fallible_windows_service_run");
 
     let rt = tokio::runtime::Runtime::new()?;
-
-    // Fixes <https://github.com/firezone/firezone/issues/4899>,
-    // DNS rules persisting after reboot
-    crate::dns_control::deactivate().ok();
```

Comment:
This was redundant, the platform-independent main loop for the IPC service already deactivates DNS before listening for the first GUI connection